SR2021/2021-02-14-league-2-followup.md

```diff
@@ -0,0 +1,34 @@
+---
+to: Teams who attended the leagues but aren't very active
+subject: Preparing for league 3
```

Comment:
But mah unicode

```suggestion
subject: Preparing for league Ⅲ
```

Reply:
Sooo, the league sessions are on the website with Arabic numerals. It's the rules' modules which use the roman numerals. I'm therefore tempted to leave this as is.

Reply:
Probably something more like "Need extra support?" over directly mentioning mentors in the subject.